frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/create-vm-wizard.tsx

```diff
@@ -406,6 +407,16 @@ export const CreateVMWizardPageComponent: React.FC<CreateVMWizardPageComponentPr
         namespace: activeNamespace,
         prop: VMWizardProps.dataVolumes,
       }),
+      {
```

Comment:
nads do not have to be loaded for vm import

Reply:
no nads for import :-)